rust/Cargo.toml

```diff
@@ -1,15 +1,24 @@
+[workspace]
+members = ["driver"]
+
 [package]
 name = "evmrs"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+# workaround for enabling mock feature also in integration tests
+mock = ["dep:mockall"]
+
 [dependencies]
 bnum = "0.11.0"
 evmc-vm = { git = "https://github.com/Fantom-foundation/evmc", branch = "tosca-extensions" }
+mockall = { version = "0.13.0", optional = true }
 sha3 = "0.10.8"
 
 [dev-dependencies]
```

Comment:
integration tests use dev-dependecies? how does this work?

Reply:
yes. tests and benchmarks use additionally to the normal dependencies also dev-dependecies.